docs/network_chaos.md

```diff
@@ -0,0 +1,26 @@
+### Network chaos
+Scenario to introduce network latency, packet loss, bandwidth restriction in the Node's hostnework interface. The purpose of this scenario is to observe faults caused by random variations in the network.
+
+##### Sample scenario config
+```
+network_chaos:                                    # Scenario to create an outage by simulating random variations in the network.
+  duration: 300                                   # in seconds - during with network chaos will be applied.
+  node_name:                                      # comma separated node names on which scenario has to be injected.
+  label_selector: node-role.kubernetes.io/master  # when node_name is not specified, a node with matching label_selector is selected for running the scenario.
+  instance_count: 1                               # Number of nodes to execute network chaos in.
+  interfaces:                                     # List of interface on which to apply the network restriction.
+  - "ens5"
+  execution: serial|parallel                      # Execute each of the egress option as a single scenario(parallel) or as separate scenario(serial).
+  egress:
+    latency: 50ms
+    loss: 0.02                                    # percentage
+    bandwidth: 100mbit
+```
+
+##### Steps
```

Comment:
Might be helpful here to add in a section about where to find the interface name and if it fails where to search similar to [here](https://github.com/cloud-bulldozer/kraken/blob/master/docs/zone_outage.md#debugging-steps-in-case-of-failures)

Reply:
will do,  will add instructions on how to find the interface name. Did add a hint on how an interface is chosen , if the user does not provide an interface.
 "Verify interface list in one of the node or use the interface with default route, as test interface, if no interface is specified by the user."

Will try to be more explicit.

Reply:
Think that's good then. Thought we had to go to aws to get the interface. Never mind this comment